project/ws/viewer/src/lib/plugins/practice/components/overview/overview.component.ts

```diff
@@ -106,11 +110,20 @@ export class OverviewComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   checkForAssessmentSubmitAlready(identifier: any) {
-    this.quizSvc.canAttend(identifier).subscribe(response => {
-      if (response && response.attemptsMade > 0) {
-        this.quizSvc.checkAlreadySubmitAssessment.next(true)
-      }
-    })
+    if (this.selectedAssessmentCompatibilityLevel < 6) {
+      this.quizSvc.canAttend(identifier).subscribe(response => {
+        if (response && response.attemptsMade > 0) {
+          this.quizSvc.checkAlreadySubmitAssessment.next(true)
+        }
+      })
+    } else {
+      this.quizSvc.canAttendV5(identifier).subscribe(response => {
```

Comment:
This condition part should not be here, please check with latest pull from cbrelease-4.8.15-release branch